test/corpus/json.txt

```diff
@@ -0,0 +1,47 @@
+================================================================================
+JSON traversal
+================================================================================
+
+SELECT
+    "user"->>'login' as login,
+    "user"->'address' ->> 'city',
+    "user"->'address' ->> 'state' as userstate,
+    "user" #> 'items',
+    "user" #>> 'more_items' AS more
+FROM users;
+
+--------------------------------------------------------------------------------
+
+(program
+ (statement
+  (select
+   (keyword_select)
+   (select_expression
+    (term
+     (json_traversal
+      (literal))
+     (keyword_as)
+     (identifier))
+    (term
+     (json_traversal
+      (json_traversal
+       (literal))))
```

Comment:
This subtree represents the term `"user"->'address' ->> 'city'`. It only references a single `literal` node which means the other two have been hidden.

I've puts together https://github.com/DerekStride/tree-sitter-sql/pull/171 to show an alternative using the `binary_expression` node that will surface all 3 literals and treat the navigation nodes as anonymous operators.